### PR TITLE
K8SPXC-1333: Include namespace to scheduled backup prefix

### PR DESCRIPTION
**Pull request by egegunes**
[K8SPXC-1333](https://jira.percona.com/browse/K8SPXC-1333) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
#1544 

**Cause:**
When operator runs in cluster wide mode, it mixes up scheduled backups if cluster names are the same in multiple namespaces because it only uses cluster name as the prefix.

**Solution:**
Include namespace to scheduled backup prefix.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?
- [x] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Are the manifests (crd/bundle) regenerated if needed?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported PXC version?
- [x] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

File: pkg/controller/pxc/backup.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileBackups(ctx context.Context, cr
 	log := logf.FromContext(ctx)
 
 	backups := make(map[string]api.PXCScheduledBackupSchedule)
-	backupNamePrefix := backupJobClusterPrefix(cr.Name)
+	backupNamePrefix := cr.Namespace + "/" + backupJobClusterPrefix(cr.Name)
```
Comment (hors) [inline pkg/controller/pxc/backup.go:37]: I think it will not work `not exceed 22 characters`. Prefix can be too long.

**Review by hors — CHANGES_REQUESTED** [review]

File: pkg/controller/pxc/backup.go (hunk 1 above)
Comment (hors) [reply pkg/controller/pxc/backup.go:37]: And this one:
`2023-12-11T18:45:28.109Z	ERROR	failed to create backup	{"controller": "pxc-controller", "namespace": "pxc1", "name": "cluster111111111222222", "reconcileID": "cb1268c3-34e5-4c88-a8f0-be64f93196f4", "error": "PerconaXtraDBClusterBackup.pxc.percona.com \"cron-cluster111111111-fs-pvc-20231211184528-1tfsi\" is invalid: metadata.labels: Invalid value: \"pxc1/ca65c-daily-backup\": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')"}`

Comment (hors) [inline pkg/controller/pxc/backup.go:37]:
```suggestion
	backupNamePrefix := cr.Namespace + "-" + backupJobClusterPrefix(cr.Name)
```

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/auto-tuning-8-0.log) |
| cross-site-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/limits-8-0.log) |
| monitoring-2-0-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/pitr-8-0.log) |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/pitr-gap-errors-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/scaling-8-0.log) |
| scheduled-backup-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/bde6d79a/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1549/commits/bde6d79a8492ed99a567f26c1ade0ed915a45407
image: `perconalab/percona-xtradb-cluster-operator:PR-1549-bde6d79a`

File: pkg/controller/pxc/backup.go (hunk 1 above)
Comment (egegunes) [reply pkg/controller/pxc/backup.go]: https://github.com/percona/percona-xtradb-cluster-operator/pull/1549/commits/a88eb7b2896b8d985a3ca0f65330bf7c9ac3705b

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/auto-tuning-8-0.log) |
| cross-site-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/limits-8-0.log) |
| monitoring-2-0-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/pitr-8-0.log) |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/pitr-gap-errors-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1549/commits/a88eb7b2896b8d985a3ca0f65330bf7c9ac3705b
image: `perconalab/percona-xtradb-cluster-operator:PR-1549-a88eb7b2`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/affinity-8-0.log) |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/auto-tuning-8-0.log) |
| cross-site-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/cross-site-8-0.log) |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/demand-backup-cloud-8-0.log) |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/demand-backup-encrypted-with-tls-8-0.log) |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/demand-backup-8-0.log) |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/haproxy-5-7.log) |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/haproxy-8-0.log) |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/init-deploy-5-7.log) |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/init-deploy-8-0.log) |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/limits-8-0.log) |
| monitoring-2-0-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/monitoring-2-0-8-0.log) |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/one-pod-5-7.log) |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/one-pod-8-0.log) |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/pitr-8-0.log) |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/pitr-gap-errors-8-0.log) |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/proxy-protocol-8-0.log) |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/proxysql-sidecar-res-limits-8-0.log) |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/recreate-8-0.log) |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/restore-to-encrypted-cluster-8-0.log) |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scaling-proxysql-8-0.log) |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scaling-8-0.log) |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scheduled-backup-5-7.log) |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/scheduled-backup-8-0.log) |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/security-context-8-0.log) |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/smart-update1-8-0.log) |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/smart-update2-8-0.log) |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/storage-8-0.log) |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/tls-issue-cert-manager-ref-8-0.log) |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/tls-issue-cert-manager-8-0.log) |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/tls-issue-self-8-0.log) |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/upgrade-consistency-8-0.log) |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/upgrade-haproxy-8-0.log) |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/upgrade-proxysql-8-0.log) |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/users-5-7.log) |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/users-8-0.log) |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1549/a88eb7b2/validation-hook-8-0.log) |
| We run 37 out of 37|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1549/commits/a88eb7b2896b8d985a3ca0f65330bf7c9ac3705b
image: `perconalab/percona-xtradb-cluster-operator:PR-1549-a88eb7b2`

**Review by hors — APPROVED** [review]

**Review by inelpandzic — APPROVED** [review]

**Pull request merged**